src/app/organizations/collection/collection.component.html

```diff
@@ -101,18 +104,28 @@ <h3 *ngIf="Dockstore.FEATURES.enableNotebooks">Workflows, Notebooks, and Tools</
             <h3 *ngIf="!Dockstore.FEATURES.enableNotebooks">Workflows and Tools</h3>
 
             <div data-cy="collectionEntry" *ngFor="let entry of collection?.entries">
-              <mat-card class="pb-3">
+              <mat-card
+                *ngIf="
+                  (entry.entryType !== entryType.Notebook && !Dockstore.FEATURES.enableNotebooks) || Dockstore.FEATURES.enableNotebooks
```

Comment:
Why do you need this `*ngIf`? If it's not a notebook, then it'll satisfy the condition both when `enableNotebooks` is true and false. If it is a notebook, then `enableNotebooks` should be true, which also satisfies the condition

Reply:
> If it is a notebook, then `enableNotebooks` should be true

Yeah the thing is, this entry being a `notebook` and `enableNotebooks` being set are independent of each other, the issue was without this `*ngIf` entries that were notebooks we being shown regardless of the value of `enableNotebooks` (i.e. all the time)

The conditions here are a bit hard to imagine:

The first part:
`entry.entryType !== entryType.Notebook && !Dockstore.FEATURES.enableNotebooks`

- When the feature flag is **not** enabled, display an entry only if it is **not** a notebook

The second part:
`Dockstore.FEATURES.enableNotebooks`

- When the feature flag **is** enabled, display **all** entries (including notebooks)

That being said, the logic here can be simplified and maintain the same results :) will update to:
`entry.entryType !== entryType.Notebook || Dockstore.FEATURES.enableNotebooks`

![image](https://user-images.githubusercontent.com/97123241/233179190-df8e8591-00b7-4d49-a650-2385b18172a6.png)